src/main/java/com/seleniumeasy/alertsandmodals/ProgressBarModalPage.java

```diff
@@ -0,0 +1,41 @@
+package com.seleniumeasy.alertsandmodals;
+
+import com.codeborne.selenide.Condition;
+import com.codeborne.selenide.SelenideElement;
+import com.seleniumeasy.DemoHomePage;
+
+import static com.codeborne.selenide.Selenide.$;
+
+public class ProgressBarModalPage extends DemoHomePage {
+    private final SelenideElement buttonShowSimpleDialog = $(".btn-primary"),
+            buttonDialogWithCustomMessage = $(".btn-success"),
+            buttonDialogWithCustomSettings = $(".btn-warning"),
+            modalDialog = $(".modal h3");
+
+    public ProgressBarModalPage openSimpleDialog() {
+        buttonShowSimpleDialog.click();
+        return this;
+    }
+
+    public ProgressBarModalPage openDialogWithCustomMessage() {
+        buttonDialogWithCustomMessage.click();
+        return this;
+    }
+
+    public ProgressBarModalPage openDialogWithCustomSettings() {
+        buttonDialogWithCustomSettings.click();
+        return this;
+    }
+
+    public void verifySimpleDialogIsDisplayed(Condition c) {
+        modalDialog.should(c);
+    }
+
+    public void verifyDialogWithCustomMessageIsDisplayed(Condition c) {
```

Comment:
wrong name of PA method: you can put Condition.enabled or Condition.hidden and in that case you will have inconsistency
Please update it for all methods where you pass Condition as a parameter
you can name method like 'verifyDialogWithCustomMessage'

Reply:
Rename all methods of the class in a style like 'verifyDialogWithCustomMessage'